### PR TITLE
KEYCLOAK-18869 Authentication flow binding overrides

### PR DESCRIPTION
**Pull request by Robbilie**
## Additional Information
Currently you can create custom authentication flows on a realm CRD but you cannot use them in clients.

### CODE REVIEW

**Comment by Robbilie** [comment]
@vmuzikar @abstractj @stianst  how is the process to get something approved and merged? :)
are you occasionally checking PRs or do they have to be assigned to any of you?

File: deploy/crds/keycloak.org_keycloakrealms_crd.yaml [hunk 1]
```diff
@@ -265,6 +265,11 @@ spec:
                             type: string
                           description: Client Attributes.
                           type: object
+                        authenticationFlowBindingOverrides:
+                          additionalProperties:
+                            type: string
+                          description: Authentication Flow Binding Overrides.
+                          type: object
```
Comment (vmuzikar) [inline deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: This does not belong to the Realm CRD.

Please do not make manual changes to the CRDs and always run `make code/gen`. Thank you.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@Robbilie Thank you for your contribution!

File: deploy/crds/keycloak.org_keycloakrealms_crd.yaml (hunk 1 above)
Comment (Robbilie) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: uhm i just reverted the yaml changes and ran the command and the result is identical to my changes…?

Comment (Robbilie) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: @vmuzikar can you verify this?

Comment (Robbilie) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: @vmuzikar can you verify that?

Comment (vmuzikar) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: Sorry again for the late response. I realized it is actually ok. The Client API representation is really included in the Realm CR as it is possible to define Clients there. Sorry about the confusion.

**Review by vmuzikar — APPROVED** [review]

File: deploy/crds/keycloak.org_keycloakrealms_crd.yaml (hunk 1 above)
Comment (Robbilie) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:272]: no worries, i wasnt following the proper steps in my first try so it couldve just as well been an error on my side so i verrified, so even though unintentionally, you made me follow proper procedures :P

**Pull request merged**